### PR TITLE
Disable wiki on all repositories but velocitas-docs

### PR DESCRIPTION
**Pull request by kse3hi**
As far as we document our Velocitas Project centralized in one place, to avoid maintenance effort, proposal is to disable wiki tab for other project but velocitas-docs to not confuse our user/customers.

### CODE REVIEW

**Comment by netomi** [comment]
Can I get an approval from another project member for that change?

Otherwise the PR looks good to me and have approved already.

**Review by netomi — APPROVED** [review]

**Comment by kse3hi** [comment]
@doosuu, @gkreyssing can you please approve as well?

**Comment by netomi** [comment]
I think the PR makes sense in order to disable things that are not used and might confuse people.
Maybe you should ping your colleagues, as the notification might got lost.

**Comment by kse3hi** [comment]
@BjoernAtBosch , @MP91  please review and approve.

**Comment by heurtematte** [comment]
@gkreyssing or @ariexi can you approve this PR?

File: otterdog/eclipse-velocitas.jsonnet
Comment (MP91) [inline otterdog/eclipse-velocitas.jsonnet]: @BjoernAtBosch should we archive this now?

File: otterdog/eclipse-velocitas.jsonnet
Comment (MP91) [inline otterdog/eclipse-velocitas.jsonnet:234]: is there a reason to not add it everywhere?
```suggestion
      delete_branch_on_merge: true,
      description: "license-check",
      has_wiki: false,
```

Comment (kse3hi) [reply otterdog/eclipse-velocitas.jsonnet:234]: Fixed

File: otterdog/eclipse-velocitas.jsonnet
Comment (kse3hi) [reply otterdog/eclipse-velocitas.jsonnet]: Motivation:
1. Not usage detected
2. Archiving only made repo "read-only". Currently, no maintain is expected. All functionality moved into devenv-runtimes repo

Comment (BjoernAtBosch) [reply otterdog/eclipse-velocitas.jsonnet]: If it's as easy as setting those flags to true, we should (at least for the deprecated runtime repos).

File: otterdog/eclipse-velocitas.jsonnet
Comment (BjoernAtBosch) [inline otterdog/eclipse-velocitas.jsonnet:109]: Did you check with @doosuu if this shall be archived?

File: otterdog/eclipse-velocitas.jsonnet
Comment (MP91) [reply otterdog/eclipse-velocitas.jsonnet]: yes runtime-k3d is not used anymore and should be archived so please add this flag

File: otterdog/eclipse-velocitas.jsonnet
Comment (MP91) [reply otterdog/eclipse-velocitas.jsonnet:109]: this is currently completely empty and we never used it...

If we need it we can bring it back from the archive

Comment (BjoernAtBosch) [reply otterdog/eclipse-velocitas.jsonnet:109]: Ok, if it's easy, let's do it.

File: otterdog/eclipse-velocitas.jsonnet
Comment (BjoernAtBosch) [inline otterdog/eclipse-velocitas.jsonnet]: This is no template repo, or is it?
```suggestion
```

**Review by BjoernAtBosch — CHANGES_REQUESTED** [review]

File: otterdog/eclipse-velocitas.jsonnet
Comment (MP91) [inline otterdog/eclipse-velocitas.jsonnet]:
```suggestion
      description: ".github",
```

**Review by MP91 — CHANGES_REQUESTED** [review]

**Review by MP91 — APPROVED** [review]
LGTM

File: otterdog/eclipse-velocitas.jsonnet
Comment (ariexi) [inline otterdog/eclipse-velocitas.jsonnet:294]: Do we need in this section also the attribute "has_wiki: false,"?

**Review by netomi — APPROVED** [review]

File: otterdog/eclipse-velocitas.jsonnet
Comment (ariexi) [reply otterdog/eclipse-velocitas.jsonnet:294]: @kse3hi @MP91 @BjoernAtBosch @netomi COuld you please have an eye on my "finding" before I approve the request?
Thx!

**Review by BjoernAtBosch — APPROVED** [review]
LGTM

**Comment by netomi** [comment]
changes are live.

**Comment by ariexi** [comment]
> changes are live.

But as asked in my review comment, the vehicle-app-cpp-template has still a wiki ...

**Comment by netomi** [comment]
that was missed sorry, I have disabled the wiki also for that repo and updated the config.

**Comment by ariexi** [comment]
Thx! Great!

File: otterdog/eclipse-velocitas.jsonnet
Comment (ariexi) [reply otterdog/eclipse-velocitas.jsonnet:294]: Resolved

**Review by ariexi — COMMENTED** [review]
LGTM now

**Pull request merged**
